notebooks/data-sources/oc-github-repo/github_PR_EDA.ipynb

```diff
@@ -2,7 +2,7 @@
  "cells": [
```

Comment:
<p>Should this be converted into days as well? Or is seconds appropriate for this approach? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/287/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>

Reply:
<p>In this cell we're just looking at raw data and exploring what the bins would look like, if we were to split the data into a 10 equal parts for a classification setup. So as of this cell, no decision is being made about days vs seconds (pls correct me if I'm wrong @oindrillac)</p>